docker_cli/_docker_cuda.py

```diff
@@ -0,0 +1,284 @@
+from __future__ import annotations
+
+import textwrap
+from abc import ABC, abstractmethod
+from typing import Iterable, List
+
+from ._package_manager import PackageManager, get_package_manager
+from ._url_reader import URLReader
+
+
+class CUDADockerfileGenerator(ABC):
+    """
+    An abstract base class for CUDA dockerfile generators.
+
+    CUDA dockerfile generators receive a cuda version and base image, and can
+    produce a dockerfile that contains instructions for building CUDA on the
```

Comment:
Let's capitalize "Dockerfile" and "CUDA" in public docstrings.

```suggestion
    An abstract base class for CUDA Dockerfile generators.

    CUDA Dockerfile generators receive a CUDA version and base image, and can
    produce a Dockerfile that contains instructions for building CUDA on the
```